config/locales/en/helpers.yml

```diff
@@ -140,6 +146,7 @@ en:
             'no': 'No'
       steps_case_details_form:
         ufn: Your Unique File Number
+        remitted_to_magistrate_date: label remittal date
```

Comment:
I assume this is either incorrect or not used - please update/remove

Reply:
removed